admin/proc-rhdh-deploy-aks.adoc

```diff
@@ -0,0 +1,302 @@
++
+[source]
+----
+az aks approuting enable --resource-group <your_ResourceGroup> --name <your_ClusterName>
+----
++
+[NOTE]
+====
+Once you install the Ingress Controller, the 'app-routing-system' namespace with the Ingress Controller will be deployed in your cluster. Note the address of your {product-short} application from the installed Ingress Controller (for example, 108.141.70.228) for later access to the {product-short} application.
+====
++
+[source,bash]
+----
+kubectl get svc nginx --namespace app-routing-system -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+----
+
+* *Namespace management*: You can create a dedicated namespace for {product-short} deployment in AKS using the following command:
++
+[source,bash]
+----
+kubectl create namespace <your_namespace>
+----
+--
+
+== Deploying the {product} on Azure Kubernetes Services (AKS) using Helm deployment
+
+You can use the following procedure to deploy your {product-short} on AKS using the Helm deployment.
+
+.Procedure
+
+. Open terminal and run the following command to add the Helm Chart repository:
++
+--
+[source]
+----
+helm repo add openshift-helm-charts https://charts.openshift.io/
+----
+--
+
+. To create `ImagePull Secret`, run the following command:
++
+--
+[source,bash]
+----
+kubectl -n <your_namespace> create secret docker-registry rhdh-pull-secret \
+    --docker-server=registry.redhat.io \
+    --docker-username=<redhat_user_name> \
+    --docker-password=<redhat_password> \
+    --docker-email=<email>
+----
+--
+
+. Create a file named `values.yaml` using the following template:
++
+--
+[source,yaml]
+----
+global:
+  host: <app_address>
+route:
+  enabled: false
+upstream:
+  ingress:
+    enabled: true
+    className: webapprouting.kubernetes.azure.com
+    host: 
+backstage:
+  image:
+    pullSecrets:
+      - rhdh-pull-secret
+  podSecurityContext:
+    fsGroup: 3000
+postgresql:
+  primary:
+    podSecurityContext:
+      enabled: true
+      fsGroup: 3000
+volumePermissions:
+  enabled: true
+----
+--
+
+. To install Helm Chart, run the following command:
++
+--
+[source]
+----
+helm -n <your_namespace> install -f values.yaml <your_deploy_name> openshift-helm-charts/redhat-developer-hub --version 1.1.0
+----
+--
+
+. Verify the deployment status using the following command:
++
+--
+[source,bash]
+----
+kubectl get deploy <your_deploy_name>-developer-hub -n <your_namespace>
+----
+--
+
+. . Access the deployed {product-short} using the URL: `https://<app_address>`, where <app_address> is the Ingress address obtained earlier (for example, `https://108.141.70.228`).
+. Run the following command to upgrade the deployment:
++
+--
+[source]
+----
+helm -n <your_namespace> upgrade -f values.yaml <your_deploy_name> openshift-helm-charts/redhat-developer-hub --version 1.1.0
+----
+--
+
+. Run the following command to delete the deployment:
++
```

Comment:
Nitpick, would rather first explain when to use the delete command:
```suggestion
. To delete the deployment, run the following command :
+
```

Reply:
Done